common/metrics.md

```diff
@@ -6,24 +6,24 @@ To provide full [3D Observability](https://www.wavefront.com/wavefront-enhances-
 * Errors – the number of failed requests per second.
 * Duration – per-minute histogram distributions of the amount of time that each request takes.
 
-The `WavefrontTracer` derives the RED metrics from your spans automatically, with no additional configuration or instrumentation on your part. To visualize these out-of-the-box metrics and histograms in Wavefront:
+The `WavefrontTracer` derives the RED metrics from your spans automatically, with no additional configuration or instrumentation on your part. To visualize the out-of-the-box metrics and histograms in the default Wavefront Service Dashboard:
 
-1. Select **Applications -> Inventory** in the Wavefront task bar to display the Application Services page. If necessary, scroll to find your application and its services.
-2. Click on the service you want to see metrics for. If necessary, click on **Overall** to expand the section.
+1. Select **Applications -> Service Dashboard** in the Wavefront taskbar. 
```

Comment:
Usually, we just use > and not ->. Sorry I didn't catch this before. 

Also I call it task bar (not taskbar)